hstream-sql/src/HStream/SQL/Codegen.hs

```diff
@@ -251,9 +252,13 @@ genStreamWithSourceStream taskName frm = do
               throwSQLException CodegenException Nothing "Impossible happened"
 
 genTaskName :: IO Text
-genTaskName = do
-  MkSystemTime time _ <- getSystemTime'
-  return $ pack $ showHex time ""
+-- Please do not encode the this id to other forms,
+-- since there is a minor issue related with parsing.
+-- When parsing a identifier, the first letter is required to be a letter.
+-- When parsing a string, quotes are required.
+-- Currently there is no way to parse an id start with digit but contains letters/
+genTaskName = pack . show <$> genUnique
```

Comment:
I was pondering for a long time about that do we really need always bring randomness in the task name generation.

Reply:
Do you have other plans? if you do please present the plan.

Reply:
I thought we should allow task name without timestamp.

Reply:
https://github.com/hstreamdb/hstream/issues/414 Welcome to discuss.